.github/workflows/_test:additional-checks.yaml

```
@@ -0,0 +1,36 @@
+name: 🧪 additional-checks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - .github/workflows/_test:additional-checks.yaml
+      - .github/actions/additional-checks/**
+      - __tests__/fixtures/additional-checks*
+
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  ddev-alias:
+    runs-on: small-al2023
```

Our self-hosted runners don't run on OSS repos, we'll need to update this to `ubuntu-latest`


```suggestion
    runs-on: ubuntu-latest
```

This would be the reason for the tests not running